### PR TITLE
feat: changes on Embed APIs

### PR DESCRIPTION
**Pull request by tl-genie**
## Description

- Added new Embed APIs
- Applied changes regarding to Embed APIs

## Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Others

## Checklist

Before you submit this PR, please make sure you have completed the following:

- [x] I have read the [CONTRIBUTING.md](https://github.com/twelvelabs-io/twelvelabs-python/blob/main/CONTRIBUTE.md) document.
- [x] I have ensured my PR title is descriptive and in imperative mood.
- [x] I have added necessary documentation (if appropriate).

## Further comments

Example code runs fine.

image
image
image

### CODE REVIEW

**Review by henry-hs-choi — APPROVED** [review]
LGTM 🚀

**Review by henry-hs-choi — APPROVED** [review]

File: src/models/embed/index.ts [hunk 1]
```diff
@@ -1,70 +1,96 @@
...
 
 export interface Embedding {
-  float: number[];
+  segments?: SegmentEmbedding[];
+  errorMessage?: string;
+  metadata?: EmbeddingMediaMetadata;
+}
+
+export interface SegmentEmbedding {
+  float?: number[];
```
Comment (park-yeonhoo) [inline src/models/embed/index.ts:20]: embedding_floats

Comment (tl-genie) [reply src/models/embed/index.ts:32]: to make it consistent with Py SDK, I think it's better to have a same name `embeddingFloat`

Comment (park-yeonhoo) [reply src/models/embed/index.ts:32]: sorry EmbeddingsFloat

**Review by park-yeonhoo — APPROVED** [review]

**Pull request merged**